hardware/OpenWebNetTCP.cpp

```diff
@@ -895,14 +905,14 @@ bool COpenWebNetTCP::sendCommand(bt_openwebnet& command, vector<bt_openwebnet>&
 /**
     automatic scan of automation/lighting device
 **/
-void COpenWebNetTCP::scan_automation_lighting()
+void COpenWebNetTCP::scan_automation_lighting(int cen_area)
```

Comment:
i think this can be const
`scan_automation_lighting(const int cen_area)`

Maybe with more functions?